pkg/tuned/controller.go

```diff
@@ -801,19 +978,21 @@ func (c *Controller) changeSyncerTuneD(change Change) (synced bool, err error) {
 		}
 		reload = reload || changeProvider
 
-		if c.daemon.recommendedProfile != change.recommendedProfile {
+		if (c.daemon.recommendedProfile != change.recommendedProfile) || change.nodeRestart {
+			// trigger a file write when the deferred flag is removed from
+			// the existing profile (edit) so we can update the status
```

Comment:
I'm not sure I understand the "(edit)".  Can you expand/explain or just leave it out if it makes sense?  The operator creates/updates Profiles. Edit to me suggests administrator action on Profiles which the administrator does not do.  Admin edits Tuned CRs.

Reply:
turns out the comment is now obsolete, removed.